mkdocs/docs/HPC/setting_up_python_virtual_environments.md

```diff
@@ -0,0 +1,221 @@
+# Python Virtual Environments (venv's)
+
+## Introduction
+
+Python virtual environments (venv's) are a way to create isolated Python environments.
+You can install packages in a venv without affecting the system Python installation.
+Because a normal user cannot install packages globally, 
+venv's are a way to install packages without needing root access.
+This is useful when you need to use a package that is not available as a module on the HPC cluster.
+
+## Creating a venv
+
+A venv can be created with the following commands:
+
+```bash
+$ python -m venv myenv      # Create a new venv named myenv
+$ source myenv/bin/activate # Activate the venv
+```
+
+After activating the venv, you can install packages with `pip`:
```

Comment:
I would avoid using `venv`, use `Python virtual environment` instead.

Also explain better what "activating" does, make a separate subsection for that (create, activate, install, use)

Reply:
I have replaced some occurrences of `venv` with `python virtual environment`. Other occurrences I just changed to `virtual environment`. See [33ac8a2](https://github.com/hpcugent/vsc_user_docs/pull/639/commits/33ac8a20833fe1dcca543de078d03d72a4a89e7f)